Jenkinsfile

```diff
@@ -393,22 +398,25 @@ pipeline {
                     source \${PWD}/.envrc
                     set -x
 
+                    suffix=""
+                    if [ "$USE_SLE_BASE" == "true" ]; then
+                        OS="sle"
+                    else
+                        OS="opensuse"
+                        suffix="-opensuse"
+                    fi
+
                     kubectl delete storageclass hostpath || /bin/true
                     kubectl create -f - <<< '{"kind":"StorageClass","apiVersion":"storage.k8s.io/v1","metadata":{"name":"hostpath"},"provisioner":"kubernetes.io/host-path"}'
 
                     # Unzip the bundle
                     rm -rf output/unzipped
                     mkdir -p output/unzipped
-                    unzip -e output/scf-*amd64*.zip -d output/unzipped
+                    unzip -e output/scf-$OS-*.zip -d output/unzipped
```

Comment:
This is in a `"""` block, so `${}` is being interpreted by Jenkins instead of `sh`. You need `\${OS}` here.

Reply:
THX. Ok, that explains why the params. variable worked in one case and not the other.